src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -23,6 +23,30 @@
   }
 }
 
+.AddonReviewCard-addonReviewLabel {
+  font-weight: normal;
+  margin-left: 10px;
+}
+
+.AddonReviewCard-addonNameBold {
+  font-weight: bold;
+}
```

Comment:
`.AddonReviewCard-addonName` seems descriptive enough - avoid hardcoding the style properties in the name.